core/server/users.js

```diff
@@ -176,6 +176,16 @@ Meteor.users.find({ 'status.online': true }).observeChanges({
     if (!user.status.lastLogoutAt) return;
 
     const { defaultLevelId, respawnDelay } = Meteor.settings;
+    const { guildId } = user;
+    const guild = Guilds.findOne(guildId);
```

Comment:
I think we need to think of another solution.

- Not all users are in a guild so for users without a guild it will crash
- You are server side so lp.notif.warning doesn't exist here